docs/api/text/postag.rst

```diff
@@ -223,6 +223,58 @@ Libraries like SpaCy use tags from the Universal Dependencies (UD) framework. Th
 | WRB               | Wh-adverb                                |                      |                          |
 +-------------------+------------------------------------------+----------------------+--------------------------+
 
+Parsing raw text automatically
+---------------------------
+
+The ``PosTagVisualizer`` can also be used with untagged text by using the ``parse`` keyword on instantiation. The keyword
+to parse indicates which natural language processing library to use. To use ``spacy``:
+
+
+.. code:: python
+
+    untagged_speech = u'Whose woods these are I think I know'
+
+    # Create the visualizer, fit, score, and show it
+    viz = PosTagVisualizer(parser='spacy')
+    viz.fit(untagged_speech)
+    viz.show()
+
+
+Or, using the ``nltk`` parser.
+
+.. code:: python
+
+    untagged_speech = u'Whose woods these are I think I know'
+
+    # Create the visualizer, fit, score, and show it
+    viz = PosTagVisualizer(parser='nltk')
+    viz.fit(untagged_speech)
+    viz.show()
+
+.. note::
+    To use either of these parsers, either `nltk` or `spacy` must already be installed in your environment.
+
+You can also change the tagger used. For example, using `nltk` you can select either `word` (default):
+
+.. code:: python
+
+    untagged_speech = u'Whose woods these are I think I know'
+
+    # Create the visualizer, fit, score, and show it
+    viz = PosTagVisualizer(parser='nltk', tagger='word')
+    viz.fit(untagged_speech)
+    viz.show()
+
+Or using `wordpunct`.
+
+.. code:: python
+
+    untagged_speech = u'Whose woods these are I think I know'
+
+    # Create the visualizer, fit, score, and show it
+    viz = PosTagVisualizer(parser='nltk', tagger='wordpunct')
+    viz.fit(untagged_speech)
+    viz.show()
 
```

Comment:
Well written and clear updates to the documentation, thank you!

Reply:
Thanks!